exchanges/orderbook/tranches.go

```diff
@@ -18,7 +18,7 @@ var (
 	errInvalidNominalSlippage          = errors.New("invalid slippage amount, its value must be greater than or equal to zero")
 	errInvalidImpactSlippage           = errors.New("invalid slippage amount, its value must be greater than zero")
 	errInvalidSlippageCannotExceed100  = errors.New("invalid slippage amount, its value cannot exceed 100%")
-	errBaseAmountInvalid               = errors.New("invalid base amount")
+	errSnapshotAmountInvalid           = errors.New("invalid snapshot amount")
```

Comment:
🚧 Was right the first time
```suggestion
	errBaseAmountInvalid               = errors.New("invalid base amount")
```